Symfony/CS/Tests/Fixer/Contrib/MultipleUnsetFixerTest.php

```diff
@@ -0,0 +1,116 @@
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author SpacePossum
+ */
+class MultipleUnsetFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return array(
+            // spaces_after_semicolon
+            array(
+                '<?php UNSET($a,$b,$c/**/); ',
+                '<?php UNSET($a); unset($b,$c/**/);',
+            ),
+            array(
+              '<?php
+              $config = array();
+              if ($config) {
+              }
+              unset($config[\'autoescape_service\'], $config[\'autoescape_service_method\']);
+              ',
+            ),
+            array(
+                '<?php unset($d , $e);/*unset(    $d2);unset($e   );;*/',
+                '<?php unset($d );/*unset(    $d2);unset($e   );;*/uNseT( $e);',
+            ),
+            // whitespacy_lines
+            array(
+                '<?php //1
+                    unset($foo/*;*/,/*;*/$bar, $c ,$foobar  ,  $foobar2);
+                     //test
+                     /* more comment test*/
+                    '.'
+                ',
+                '<?php //1
+                    unset($foo/*;*/);
+                    unset(/*;*/$bar, $c ); //test
+                    unset($foobar  ); /* more comment test*/
+                    unset(  $foobar2);
+                ',
+            ),
+            // trailing_spaces
+            array(
+                '<?php //2
+                    unset($foo,$bar,$foobar,$foobar2,$foobar3);/*1*/
+                    /*2*/
+                    //3
+                    /*4*/
+                    /*5*/ '.'
+                ',
+                '<?php //2
+                    unset($foo);/*1*/
+                    unset($bar);/*2*/
+                    unset($foobar);//3
+                    unset($foobar2);/*4*/
+                    /*5*/ unset($foobar3);
+                ',
+            ),
+            // extra_empty_lines
+            array(
+                '<?php
+                    unset($foo3,$bar,$test,$test1);
+                        /* c1 */
+                        '.'
+                '.'
+                // c2
+                '.'
+                ',
+                '<?php
+                    unset($foo3);
+                        /* c1 */
+                        unset($bar);
+                '.'
+                // c2
+                unset($test,$test1);
+                ',
+            ),
+            array(
+                '<?php (unset)$f;',
+            ),
+            array(
+                '<?php unset($x,$b  , $d);  /**/   ?> b',
+                '<?php unset($x);  /**/ unset ($b  , $d) ?> b',
+            ),
+            array(
+                '<?php unset($x) ?>',
+            ),
+            array(
+                '<?php unset($y,$u); ?>',
```

Comment:
the simplest case and it is on the end... but found it!
why without the space? we add a comma, we should add a space.